packages/create-spectacle/src/cli.ts

```diff
@@ -46,10 +46,11 @@ const main = async () => {
   let i = 0;
   let type = argv[ArgName.type] || argv['t'];
   let name = argv['_']?.[0];
-  let lang = argv[ArgName.lang] || argv['l'] || 'en';
-  let port = argv[ArgName.port] || argv['p'] || 3000;
+  let lang = argv[ArgName.lang] || argv['l'];
+  let port = argv[ArgName.port] || argv['p'];
```

Comment:
Moving these fallbacks to the prompts. Result of this...

* **before this change** user could pass `type` and `name`, and they wouldn't be prompted at all – just straight to creating decks.
* **after this change** user can pass `type` and `name`, but they'll still get prompted to confirm lang/port. If _all_ of the values are explicitly provided via flags (and no overwrite detected), prompts will be bypassed.